pkg/innerring/processors/settlement/audit/calculate.go

```diff
@@ -60,8 +60,9 @@ func (c *Calculator) Calculate(p *CalculatePrm) {
 	log.Info("calculate audit settlements")
 
 	log.Debug("getting results for the previous epoch")
+	prevEpoch := p.Epoch - 1
```

Comment:
Lets add sanity zero check?

Reply:
Added zero check at the top of the function.